tests/e2e/workloads/ocp/registry/test_registry_by_increasing_num_of_image_registry_pods.py

```diff
@@ -71,6 +75,20 @@ def test_registry_by_increasing_num_of_registry_pods(self, count=3):
 
         # Validate pvc mounted on image registry pod
         validate_pvc_mount_on_registry_pod()
+        # Coverage for Bz 2128263
```

Comment:
You are doing this verification at the very beginning and no registry ops were done here.
Should we move this coverage to the end?